pkg/blockstorage/awsefs/awsefs.go

```diff
@@ -41,7 +41,7 @@ type Efs struct {
 	accountID       string
 	region          string
 	role            string
-	backupVaultName string
```

Comment:
Why does this need to be exported?

Reply:
in `kio/aws/snapshot.go` in [this PR](https://github.com/kastenhq/k10/pull/14968), there would be the following:

> func SnapshotsListWLimit(ctx context.Context, tags map[string]string, limit int64, awsEfs *awsefs.Efs) ([]*blockstorage.Snapshot, error) {
	result := make([]*blockstorage.Snapshot, 0)
	var err error
	resp := (&backup.ListRecoveryPointsByBackupVaultOutput{}).SetNextToken("")
	req := &backup.ListRecoveryPointsByBackupVaultInput{}
	req.SetBackupVaultName(awsEfs.BackupVaultName)

Reply:
`BackupVaultName` will be reference here!